buildpacks/java/tests/setup.go

```diff
@@ -19,6 +21,33 @@ func NewLogger() bard.Logger {
 	return bard.NewLogger(buf)
 }
 
+type HTTPFunction struct {
```

Comment:
This structure made sense for python, though I'm wondering if it should be adjusted for java. For instance, the term `module` might more accurately map to a java `package` instead.

Reply:
Started down path of converting to java, but realized build.go doesn't need the Java function at all, so removed it entirely

Reply:
I see. Yeah I guess it's just needed by the lifecycle in general to stuff in the image, but this buildpack doesn't reference it directly like python does for the validation.

It does make me wonder, should the Java buildpack do some basic validation too? Namely, checking that the referenced function actually exists in the user-provided source. Not for this PR for sure, but as a separate feature.

cc: @andrew-su @bbtong